testsuite/testsuite.sh

```diff
@@ -141,7 +141,7 @@ do_gna () {
 # The Python Unit testsuite: regression testsuite for Python bindings to libghdl
 do_pyunit () {
   gstart "[GHDL - test] pyunit"
-  PYTHONPATH=$(pwd)/.. python3 -m pytest pyunit
+  PYTHONPATH=$(pwd)/.. python3 -m pytest -rA --cov=.. --cov-config=.coveragerc pyunit
```

Comment:
Do we want to enable coverage collection always? We run this test suite in 8-10 different jobs at the same time. We need to either merge all of them at the end (if possible), or discard all but one.

If we enable some external service, can it handle several submits for the same CI run?

Reply:
I think one of them can handle a "label" like nightly, regression, etc. so we could distinguish results.

In my Python projects I run only on Python 3.8 (will change to 3.9 soon), but don't run on 3.6, 3.7, ...
Uploading is currently not enabled and we have just 7 tests, so this is harmless, but yes, we should think about load reduction when the upload to the server works.

Reply:
`-rA` report all results
`--cov=.. --cov-confi=.coveragerc` is for coverage

I move this from `pytest.ini`, because paths are relative to working dir not the `pytest.ini`. I wanted to avoid confusion if parameters are in the ini, but need a call from a specific directory.

Reply:
I propose the following:

```
PYTHONPATH=$(pwd)/.. python3 -m pytest -rA $PYTEST_ARGS pyunit
```

Then, in just one of the jobs, set:

```yml
env:
  PYTEST_ARGS: '--cov=.. --cov-config=.coveragerc'
```

Reply:
See branch [ci/coverage](https://github.com/umarcor/ghdl/commits/ci/coverage) in my fork. Feel free to pick the first commit only, or both of them. See CI run: https://github.com/umarcor/ghdl/actions/runs/455551490

Reply:
Yes, I agree with @umarcor : just enable coverage on one test.

Reply:
@umarcor I integrated your changes via FF-merge.

@tgingold this PR is then ready for merge. CI results are running for this branch, but it was already proven on Unai's fork.